sample/F8/sessions

```diff
@@ -0,0 +1,1249 @@
+        },
+
+
+        "tags": ["Growth"],
+        "speakers": [{
+            "organisation": "Facebook",
+            "id": 3,
+            "name": "Brian Hale "
+        }]
+    }, {
+        "short_abstract": null,
+        "long_abstract": "null",
+        "id": 7,
+        "subtitle": "litho-a-declarative-framework-for-efficient-UIs",
+
+        "title": "Litho: A Declarative Framework for Efficient UIs",
+
+        "sessionDescription": "Components for Android allows developers to implement complex, efficient RecyclerViews through a simple declarative API. In this session we’ll discuss the function and features of this powerful framework.",
+
+        "end_time": "2017-04-18T13:20:00.000Z",
+        "microlocation": {
+            "id": 6,
+            "name": "F"
+        },
+        "start_time": "2017-04-18T12:30:00.000Z",
+        "tags": ["Engineering & Open Source"],
+         "comments": "", 
+        "language": null, 
+        "slides": null, 
+        "audio": null, 
+        "video": null, 
+        "signup_url": null, 
+        "state": "accepted", 
+        "timezone": "UTC",
+        "session_type": {}, 
+        "track": {},
+        "speakers": [{
+            "organisation": "Facebook",
+            "id": 4,
+            "name": "Lucas Rocha "
+        }, {
+            "organisation": "Facebook",
+            "id": 5,
+            "name": "Emil Sjölander"
+        }]
+    }, {
+        "short_abstract": null,
+        "long_abstract": "null",
+        "id": 8,
+        "subtitle": "webvr-enabling-developers-to-build-engaging-cross-platform-vr-content",
+
+        "title": "WebVR: Enabling Developers to Build Engaging Cross-Platform VR Content",
+
+        "sessionDescription": "Web-based frameworks such as React VR are making it easier than ever to create VR content that can be shared with a huge audience across VR, mobile, and the web. Coupled with the new web browser that Oculus is building, more of the web will become immersive over the coming years. Join us to learn what you can build with WebVR and how this new technology can help enhance your existing content.",
+
+        "start_time": "2017-04-18T12:30:00.000Z",
+        "end_time": "2017-04-18T13:20:00.000Z",
+        "microlocation": {
+            "id": 2,
+            "name": "LL"
+        },
+         "comments": "", 
+        "language": null, 
+        "slides": null, 
+        "audio": null, 
+        "video": null, 
+        "signup_url": null, 
+        "state": "accepted", 
+        "timezone": "UTC",
+        "session_type": {}, 
+        "track": {},
+        "speakers": [{
+            "organisation": "Facebook",
+            "id": 6,
+            "name": "Justin Rogers"
+        }, {
+            "organisation": "Facebook",
+            "id": 7,
+            "name": "Andrew Mo"
+        }, {
+            "organisation": "Facebook",
+            "id": 8,
+            "name": "Mike Armstrong"
+        }],
+
+
+        "tags": ["Virtual Reality"]
```

Comment:
this is a good feature. but i think then we would need a tags.json file as well to collect all the tags we have. lets see what others say on this